assignment/tool/src/webapp/vm/assignment/chef_assignments_student_preview_submission.vm

```diff
@@ -149,8 +149,15 @@
 				<img alt="$reviewIndicator" title="$reviewIndicator" src="/library/image/silk/rosette.png" class="assignmentRosette"/>
 				<em>
 					$plagiarismNote
-					#if ($plagiarismFileTypes)
-						$plagiarismFileTypes
+					#if ($plagiarismFileTypes || $plagiarismFileTypes)
```

Comment:
Same variable is twice, guess it should be plagiarismFileSize

Reply:
@josecebe, I'm still seeing the same variable twice in this conditional. As @bgarciaentornos mentioned, one of them should probably be `plagiarismFileSize`